webpack/assets/javascripts/react_app/components/notifications/helpers.js

```diff
@@ -0,0 +1,3 @@
+export const redirectToLogin = () => {
```

Comment:
nitpick: why is this a notification thing? redirect to login can happen on session timeout regardless of notification (could be triggered by any api call..)

Reply:
I agree, things like that should be handled by the API middleware, 
this was a method which was used before in the notifications component.
I wanted to add this feature as a small PR after #7222 will get merged

Reply:
opened https://github.com/theforeman/foreman/pull/7337
it is blocked by #7222 since that PR refactors almost every file of the API middleware,
@amirfefer waiting also for your PR #7186 to use the `connected-react-router` instead of `window.replace`